modules/microshift-kubeconfig-local-access.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_configuring/microshift-kubeconfig.adoc
+
+:_content-type: CONCEPT
+[id="microshift-kubeconfig-local-access_{context}"]
+= Local access kubeconfig file
+
+The local access `kubeconfig` file generates in the `/var/lib/microshift/resources/kubeadmin/kubeconfig` directory. This `kubeconfig` file provides access to the API server using `localhost`. Choose this file when you are connecting the cluster locally.
+
+.Example contents of `kubeconfig` for local access
+[source, YAML]
+----
+clusters:
+- cluster:
+    certificate-authority-data: <base64 CA>
+    server: https://localhost:6443
+----
+
+When the cluster running on the {product-title} host connects to the API server locally, the certificate authority (CA) used for validation lists `localhost` as the only Subject Alternative Name (SAN). This is true whether a `config.yaml` exists or built-in values are being used. The main IP address of the host is the default value and cannot be changed. The CA of the local access `kubeconfig` file cannot be used outside of the {product-title} host.
```

Comment:
Trying to simplify and remove some implementation details to make the point a bit clearer.

```suggestion
The localhost kubeconfig file can only be used from a client connecting to the API server from the same host. The certificates in the file do not work for remote connections.
```